lardataobj/Simulation/classes_def.xml

```diff
@@ -104,8 +104,6 @@
  <class name="std::vector<sim::OpDetBacktrackerRecord>"/>
  <class name="std::vector<sim::SupernovaTruth>"/>
  <class name="std::vector<sim::SimDriftedElectronCluster>"/>
- <class name="std::pair<int,std::set<int>>"/>
```

Comment:
Why did you remove the `std::pair<int, std::set<int>>` dictionary?

Reply:
No idea, great question! I think that's just sloppy committing, sorry!